src/loader.c

```diff
@@ -622,6 +623,7 @@ load_sixel(unsigned char        /* out */ **result,
     int colors;
     int i;
 
+fprintf(stderr, "SIZE: %d\n", size);
```

Comment:
You probably want to remove this before merging (there are other fprintf lines as well, just grep for them)